.gitignore

```diff
@@ -12,3 +12,4 @@
 *.test
 tools/functional-tester/docker/bin
 hack/tls-setup/certs
+.idea
```

Comment:
I'm not sure adding it to .gitignore is a good idea or not. But if you want to do this, could you separate a commit and create another PR for the change? It is not related to auth unit tests.